ivy/functional/frontends/tensorflow/__init__.py

```diff
@@ -1,5 +1,4 @@
 # flake8: noqa
-# flake8: noqa
```

Comment:
Please make sure that no parts of the code which isn't related to the task is changed. It causes conflicts and errors especially while performing lint tests 😀